### PR TITLE
[ALLUXIO-2072] Support group/owner in object stores

### PR DESCRIPTION
**Pull request by calvinjia**
https://alluxio.atlassian.net/browse/ALLUXIO-2072

### CODE REVIEW

**Comment by ifcharming** [comment]
LGTM, thanks for the fix!

**Comment by jsimsa** [comment]
LGTM

File: underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java [hunk 1]
```diff
@@ -347,13 +347,13 @@ public void setMode(String path, short mode) throws IOException {}
   // Not supported
```
Comment (apc999) [inline underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: should we remove this comment?
(and the following)?

Comment (calvinjia) [inline underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: I think the comment is still accurate. We do not propagate any user/group/mode info from gcs. This PR changes the default to an empty value which is accepted as undefined from a null value which would be invalid.

Comment (apc999) [inline underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: I see, could you elaborate in the comment here to make this clear? i.e., we always return empty strings here are as owner/group, and propagating no more info from gcs/s3/etc

**Comment by apc999** [comment]
left one minor suggestion on comment. otherwise it looks good to me. feel free to merge it

File: underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java (hunk 1 above)
Comment (calvinjia) [inline underfs/gcs/src/main/java/alluxio/underfs/gcs/GCSUnderFileSystem.java]: Makes sense, clarified. Thanks!

**Pull request merged**